src/compas_fab/ghpython/components/attached_collision_mesh.py

```diff
@@ -0,0 +1,97 @@
+import Grasshopper
+import System
+from ghpythonlib.componentbase import dotnetcompiledcomponent as component
+
+from compas_rhino.geometry import RhinoMesh
+from compas_fab.ghpython.components.icons import attached_collision_mesh_icon
+from compas_fab.robots import AttachedCollisionMesh
+from compas_fab.robots import CollisionMesh
+
+
+class AttachedCollisionMeshComponent(component):
+    def __new__(cls):
+        return Grasshopper.Kernel.GH_Component.__new__(cls,
+                                                       "Attached Collision Mesh",
+                                                       "Attached Collision Mesh",
+                                                       """Add an attached collision mesh to the robot.""",
+                                                       "COMPAS FAB",
+                                                       "Scene")
+
+    def get_ComponentGuid(self):
+        return System.Guid("9d995079-5068-4118-9e2e-462f34154b53")
+
+    def SetUpParam(self, p, name, nickname, description):
+        p.Name = name
+        p.NickName = nickname
+        p.Description = description
+        p.Optional = True
+
+    def RegisterInputParams(self, pManager):
+        p = Grasshopper.Kernel.Parameters.Param_GenericObject()
+        self.SetUpParam(p, "scene", "scene", ":class:`compas_fab.robots.PlanningScene` The planning scene.")
```

Comment:
Do these strings really support the RST syntax of `:class:`?

Reply:
no, it doesn't